SparseGrids/tsgHiddenExternals.hpp

```diff
@@ -30,6 +30,8 @@
 
 #ifndef __TASMANIAN_SPARSE_GRID_HIDDEN_INTERNALS_HPP
 #define __TASMANIAN_SPARSE_GRID_HIDDEN_INTERNALS_HPP
+#include <complex>
+using std::complex;
```

Comment:
* *Never* force a namespace like that, all headers that include this would now work in the `std::complex` namespace
* if you need a variable from `std::complex`, then use that explicitly
```
using std::complex <some symbol>
```
* sometimes is it better to keep explicitly including from the namespace rather than contaminating your list of symbols